src/btree/bt_vrfy.c

```diff
@@ -586,8 +661,40 @@ __verify_tree(
             WT_RET(__verify_page_content_leaf(session, ref, addr_unpack, vs));
             break;
         }
+
+        if (vs->dump_tree_shape) {
+            printf("  type: %s",
+              page->dsk->type < WT_ELEMENTS(__page_types) ? __page_types[page->dsk->type] :
+                                                            "UNKNOWN");
+            /* see __wti_page_inmem */
+            switch (page->dsk->type) {
+            case WT_PAGE_COL_FIX:
+            case WT_PAGE_COL_VAR:
+            case WT_PAGE_COL_INT:
+                printf("  entries: %" PRIu32, page->dsk->u.entries);
+                break;
+            case WT_PAGE_ROW_INT:
+                printf("  entries: %" PRIu32, page->dsk->u.entries / 2);
+                break;
+            case WT_PAGE_ROW_LEAF:
```

Comment:
Why we dump more data only for row leaf?

Reply:
This code is borrowed from the `page_inmem` function. This is because of the way the values are encoded.

Reply:
... but I think, I could just use the page's `entries` field as this calculation has already been made by `page_inmem`.